charts/prefect-server/templates/server-deployment.yaml

```diff
@@ -91,6 +91,8 @@ spec:
               value: {{ .Values.server.debug | quote }}
             - name: PREFECT_LOGGING_SERVER_LEVEL
               value: {{ .Values.server.loggingLevel | quote }}
+            - name: PREFECT_SERVER_API_HOST
```

Comment:
removed in https://github.com/PrefectHQ/prefect-helm/pull/450, the api host needs to be set to `0.0.0.0` to allow external connections

Reply:
it has to be externally accessible if they want to be able to connect a worker to it -- I can't envision a circumstance when that wouldn't be the case. @desertaxle - thoughts?

Reply:
I figure you'd control the accessibility via the ingress or something like that. I can think of a reason that you'd want the host to be `127.0.0.1` when deploying via the Helm chart.